packages/template-retail-react-app/app/pages/product-detail/index.jsx

```diff
@@ -111,23 +115,27 @@ const ProductDetail = ({category, product, isLoading}) => {
             status: 'error'
         })
     }
-    const handleAddToCart = async (variant, quantity) => {
+
+    const handleAddToCart = async (productSelectionValues) => {
```

Comment:
I modified this handler to work with both multiple items. The value passed to this function is an array of {product, variant, quantity} objects.

If the items are successfully added to the cart, we when return this object so that it can be passed to the modal. (Alternatively we could refactor the model from the productview into the page, but I didn't want to go down that rabbit hole).